android/java/AndroidManifest.xml

```diff
@@ -155,13 +155,6 @@
     android:noHistory="true"
     android:launchMode="singleTask"/>
 
-<activity android:name="com.android.billingclient.api.ProxyBillingActivity"
```

Comment:
Will we have different look for this activity now?

Reply:
we were mainly adding this ref to set export false as previous version of deps, they were exporting the activity. so we are good here.